### DISCUSSION TITLE
Use CF launcher's API key

### DISCUSSION THREAD

**Discussion by cootshk** [PrismLauncher/PrismLauncher#1750] [Ideas]
Maybe have this as an option in the launcher settings
i have the CF launcher's api key if needed
using it allows downloading curseforge packs without downloading mods manually

**Comment by Trial97** [comment]
Well, the CF API already requires an API key, so changing it will have no effect.
The manual mod download is because CF doesn't return the download link in the API for them(blocking third-party launchers).
